src/pretix/control/forms/vouchers.py

```diff
@@ -269,16 +272,16 @@ class VoucherBulkForm(VoucherForm):
         label=_("Subject"),
         widget=forms.TextInput(attrs={'data-display-dependency': '#id_send'}),
         required=False,
-        initial=_('Your voucher for {event}')
+        initial=gettext_noop('Your voucher for {event}')
```

Comment:
I do not think we need noop here as we do not want to use that text as literal text lateron. What we actually do is force-convert the lazy-translation string to a fixed translated string in `__init__`.

```suggestion
        initial=_('Your voucher for {event}')
```